ReleaseNotes.md

```diff
@@ -1,3 +1,35 @@
+### New in 0.23.0 (released 06/10/2016)
+
+**Features**
+
+ - Added support for probing for an GitHub Enterprise instance - #1404 via @haacked
```

Comment:
Maybe this should say "Added support to test whether a URL points to a GitHub Enterprise instance"